bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -461,6 +461,14 @@
 		</state>
 	</channel-type>
 
+	<channel-type id="zoneAlarmTimestamp">
```

Comment:
You could adds `<category>Time</category>`

Reply:
You have to add it at the right place inside <channel-type>. Look at an example in existing code.